[2023-12-27T22:39:40.472+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: KOH.get_data manual__2023-12-27T17:39:36.582967+00:00 [queued]>
[2023-12-27T22:39:40.481+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: KOH.get_data manual__2023-12-27T17:39:36.582967+00:00 [queued]>
[2023-12-27T22:39:40.481+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-27T22:39:40.543+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): get_data> on 2023-12-27 17:39:36.582967+00:00
[2023-12-27T22:39:40.549+0500] {standard_task_runner.py:57} INFO - Started process 5686 to run task
[2023-12-27T22:39:40.554+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'KOH', 'get_data', 'manual__2023-12-27T17:39:36.582967+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/text.py', '--cfg-path', '/tmp/tmp5xh5rlcg']
[2023-12-27T22:39:40.565+0500] {standard_task_runner.py:85} INFO - Job 165: Subtask get_data
[2023-12-27T22:39:40.668+0500] {task_command.py:416} INFO - Running <TaskInstance: KOH.get_data manual__2023-12-27T17:39:36.582967+00:00 [running]> on host vm2.myguest.virtualbox.org
[2023-12-27T22:39:40.809+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Admin' AIRFLOW_CTX_DAG_ID='KOH' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-27T17:39:36.582967+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-27T17:39:36.582967+00:00'
[2023-12-27T22:39:40.810+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-27T22:39:40.811+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/kda/ML_lab3/MLops_lab3/datasets && gdown 1X-mGhWO8DwPLljP1e0gkdAloD-GL8SLq']
[2023-12-27T22:39:40.830+0500] {subprocess.py:86} INFO - Output:
[2023-12-27T22:39:42.249+0500] {subprocess.py:93} INFO - Downloading...
[2023-12-27T22:39:42.249+0500] {subprocess.py:93} INFO - From: https://drive.google.com/uc?id=1X-mGhWO8DwPLljP1e0gkdAloD-GL8SLq
[2023-12-27T22:39:42.249+0500] {subprocess.py:93} INFO - To: /home/kda/ML_lab3/MLops_lab3/datasets/Sport_car_price.csv
[2023-12-27T22:39:42.300+0500] {subprocess.py:93} INFO -   0%|          | 0.00/49.2k [00:00<?, ?B/s]100%|##########| 49.2k/49.2k [00:00<00:00, 7.44MB/s]
[2023-12-27T22:39:42.340+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-27T22:39:42.362+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=KOH, task_id=get_data, execution_date=20231227T173936, start_date=20231227T173940, end_date=20231227T173942
[2023-12-27T22:39:42.392+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-27T22:39:42.412+0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
